### PR TITLE
Domonique Griffin Checkpoint-02

### PR DESCRIPTION
**Pull request by griffind2009**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,5 +11,8 @@
     <div id="yellow" class="bulb"></div>
     <div id="green" class="bulb"></div>
   </div>
+   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```
Comment (amaseda) [inline index.html:14]: :+1:

File: index.html [hunk 2]
```diff
@@ -11,5 +11,8 @@
...
+  <script src="script.js"></script>
```
Comment (amaseda) [inline index.html:15]: :+1:

File: index.html [hunk 3]
```diff
@@ -11,5 +11,8 @@
...
+  <script src="stoplight.js"></script>
```
Comment (amaseda) [inline index.html:16]: :+1:

File: script.js [hunk 4]
```diff
@@ -1,11 +1,27 @@
 // Create an array called `colors` that contains three colors.
+var colorArray = ['blue', 'green', 'yellow'];
```
Comment (amaseda) [inline script.js:2]: :+1:

File: script.js [hunk 5]
```diff
@@ -1,11 +1,27 @@
...
 
 // Log the last item in the array to the console.
+colorArray[2]
```
Comment (amaseda) [inline script.js:5]: :+1:

File: script.js [hunk 6]
```diff
@@ -1,11 +1,27 @@
...
+var colorArray = ['blue', 'green', 'yellow'];
 
-// Create an `instructor` object that contains three key-value pairs.
+var arrayLength = colorArray.length;
+function myFavoriteColor () {
+  for (i = 0; i< arrayLength; i++) {
+    console.log("My favorite color is " + colorArray[i])
+  }
+}
+myFavoriteColor()
```
Comment (amaseda) [inline script.js:16]: :+1: No need for a function though. Can just run the for loop!

File: script.js [hunk 7]
```diff
@@ -1,11 +1,27 @@
...
 
+// Create an `instructor` object that contains three key-value pairs.
+var instructor = {
+  name: "Raymond Reddington",
+  years: 5,
+  skils: ["JS", "HTML", "CSS"]
+}
```
Comment (amaseda) [inline script.js:23]: :+1:

File: script.js [hunk 8]
```diff
@@ -1,11 +1,27 @@
...
 // Add a `facial-hair` property to `instructor`.
-
+instructor["facial-hair"]=true
```
Comment (amaseda) [inline script.js:25]: :+1:

File: script.js [hunk 9]
```diff
@@ -1,11 +1,27 @@
...
 // Log the value of the `facial-hair` property to the console.
+instructor["facial-hair"]
```
Comment (amaseda) [inline script.js:27]: :+1:

File: stoplight.js [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+var greenLight = $("#green");
+var greenLight = button.on("click", function changeBackgroundColorToGreen(){
+})*/
+
+$(function() {
+$('#red').click(function(){
+    $(this).css('background', 'red');
+
+  });
+});
```
Comment (amaseda) [inline stoplight.js:18]: :+1: Your individual event listeners for changing the light color look good. You don't need to wrap it all in `($function(){ ... })` though.

Comment (amaseda) [reply stoplight.js:18]: Also, you didn't need to set the background to red. In the stylesheet, you'll notice some classes that care of that for you. You just need to make sure to toggle the correct classes on/off, depending on the current state of the stoplight.

**Comment by amaseda** [comment]
Didn't see any code for Questions 3-6 in Part II. Let me know if you have any questions about them. Happy to talk about it.

**Comment by griffind2009** [comment]
Hi Adrian, I submitted code for #3 of Part II, but not 4-6.  I do have questions about those 3.  I only got up to question #4 and did not have enough time to figure it out, if you have possible solutions for those that would be great.  Also, please let me know if you are not able to see question 3.  I commented out my first response so the actual solution is towards the bottom. Thank you

**Comment by amaseda** [comment]
@griffind2009 My mistake, meant to write 4-6. Gave you credit for 3!

Will have that quiz solution for you momentarily...

**Pull request closed without merging**